test/jdk/javax/swing/JRadioButton/8033699/bug8033699.java

```diff
@@ -170,7 +176,9 @@ private static void createAndShowGUI() {
         btnStart.requestFocus();
 
         mainFrame.setDefaultCloseOperation(JFrame.EXIT_ON_CLOSE);
-        mainFrame.setLayout(new BoxLayout(mainFrame.getContentPane(), BoxLayout.Y_AXIS));
+        mainFrame.getContentPane()
+                 .setLayout(new BoxLayout(mainFrame.getContentPane(),
+                                          BoxLayout.Y_AXIS));
```

Comment:
Alternatively, you can create another vertical box, put all the components into it instead of directly into `mainFrame`, and add this vertical box into the frame.